app/move/controllers/update/base.test.js

```diff
@@ -574,6 +574,8 @@ describe('Move controllers', function() {
     const res = {}
     let nextSpy
     beforeEach(async function() {
+      sinon.stub(controller, 'setFlash')
+      // sinon.stub(personService, 'unformat').returns({})
```

Comment:
Still needed?

Reply:
You got me - I managed to paste it back in when I split up the commits. Doh!